src/daily-quest/daily-quest.service.ts

```diff
@@ -251,4 +289,52 @@ export default class DailyQuestService {
       throw e;
     }
   };
+
+  getDailyQuestIdsAddedWithSuggestion = async (userId: string) => {
+    return (await this.getDailyQuestsAddedWithSuggestion(userId)).map(
+      (row) => row.suggestion_id!
```

Comment:
can we avoid using ! here?

Reply:
Yes, type is set to `suggestion_id: string | null` so is ok. Fixed, thanks!